warehouse/ingest-configuration/src/main/resources/config/shard-ingest-config.xml

```diff
@@ -19,6 +19,11 @@
   <value>${table.name.shard}</value>
 </property>
 
+<property>
+    <name>shard.index.create.uids</name>
```

Comment:
It feels like this property just belongs in `shard-ingest-config.xml`. Since we load *config.xml in our ingest scripts, this has the capacity to clobber the property. Like if you set it to false here but forgot to in the other files, it's not clear which one would win.

Reply:
I'll run a few experiments and let you know the results. Ideally we could set this once at the top level config, but it might e more helpful to be verbose about this property.